diff --git a/chrome/android/java/src/org/chromium/chrome/browser/customtabs/features/toolbar/CustomTabToolbar.java b/chrome/android/java/src/org/chromium/chrome/browser/customtabs/features/toolbar/CustomTabToolbar.java
index 748528d4d830b9aaca7f9fd77eede2fe509b212d..ed1b0a4fd40682ddefdb03033e0523112713d291 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/customtabs/features/toolbar/CustomTabToolbar.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/customtabs/features/toolbar/CustomTabToolbar.java
@@ -97,7 +97,7 @@ import java.util.regex.Pattern;
 /**
  * The Toolbar layout to be used for a custom tab. This is used for both phone and tablet UIs.
  */
-public class CustomTabToolbar extends ToolbarLayout implements View.OnLongClickListener {
+public class CustomTabToolbar extends org.chromium.chrome.browser.toolbar.top.BraveToolbarLayout implements View.OnLongClickListener {
     private static final Object ORIGIN_SPAN = new Object();
 
     /**
